Code/Framework/AzGameFramework/AzGameFramework/Application/GameApplication.h

```diff
@@ -40,6 +44,8 @@ namespace AzGameFramework
         // game.*.setreg instead. In non-release builds this will still load the dev user settings and the command line settings.
         void MergeSettingsToRegistry(AZ::SettingsRegistryInterface& registry) override;
         //////////////////////////////////////////////////////////////////////////
+
+        bool m_headless { false };
```

Comment:
nitpick: space before curly brace
```suggestion
        bool m_headless{ false };
```